### ISSUE TITLE
主标题设置为空串时副标题消失

### ISSUE THREAD

**Issue by skykevin674** [apache/incubator-echarts#9656]
- [ ] I am using English in this issue. 在这个 Issue 中我使用了英文（强烈建议）。

## General Questions

- [x] Required: I have read the document and examples and tried to solve it by myself. （必填）我读过了文档和教程，并且曾试图自己解决问题。
- [x] Required: I have searched for similar issues and that didn't help. （必填）我搜索过 issue 但是没有帮助。
- [x] Required: I have tried with the latest version of ECharts and still have this problem. （必填）我试过最新版本的 ECharts，还是存在这个问题。

In this issue, I have provided information with: 在这个 issue 中我提供了以下信息：
- [x] Required: issue type；（必填）issue 类型
- [x] Required: one sentence description in issue details；（必填）一句话的问题描述
- [x] Required: demo；（必填）能反映问题的例子（如果你想提问或报 bug）
- [x] Required: ECharts version；ECharts 版本

## Issue Type

- [ ] I have a question to ask about *how to use ECharts to ...*；我想提问如何使用 ECharts 实现某功能
- [x] I have a bug to report；我想要报 bug
- [ ] I have a feature to request, e.g.: *I'd like a new feature that ...*；我需要一个新功能
- [ ] I have a feature to enhance, e.g.: *The current feature should be improved in the way that ...*；我希望改进某个功能
- [ ] There's something wrong with the documents；文档有些问题
- [ ] Others, or I'm not sure which issue types to choose here；其他，或我不知道应该选什么类型

## Issue Details

title中主标题为空字符串且副标题文本颜色为#333 时，副标题消失

### Expected Behavior

副标题显示

### Online Example

 ```
option = {
    title: {
            "text":"1",
            "subtext":"14",
            "left":"center",
            "top":0,
            "textStyle":{
                "fontSize":18,
                "fontStyle":"normal",
                "fontWeight":"bolder",
                "color":"#333"
            },
            "subtextStyle":{
                "fontSize":12,
                "fontStyle":"normal",
                "fontWeight":"normal",
                "color":"#333"
            },
            "show":true
        }
};
```

## Topics

- [x] Title
- [ ] Tooltip
- [ ] Event
- [ ] Performance
- [ ] SVG
- [ ] Map
- [ ] ECharts GL
- [ ] Third-party libraries, e.g.: Vue.js, React

## Environment

- ECharts version；ECharts 版本: 4.2.0-rc2

- [ ] It happens only on certain browsers or operating systems. 对于特定浏览器或操作系统才会出现的问题，请提供相应环境信息：Chrome 71.0.3578.98  mac OS 10.12.6

**Comment by cuijian-dexter** [comment]
it is real.
you can set color to `#333333` first.

**Comment by 100pah** [comment]
Fixed in ecomfe/zrender@88b1608cc953a2cb403ac149934bc4dbfede2797
